apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx

```diff
@@ -31,6 +32,7 @@ const UsernameTextfield = (props: ICustomUsernameProps) => {
     usernameRef,
     onSuccessMutation,
     onErrorMutation,
+    user,
```

Comment:
This is causing a type error: 

`Property 'user' does not exist on type 'ICustomUsernameProps'.ts(2339)`

Reply:
Fixed it already 🤔